test/functional/bootstrap-e2e-specs.js

```diff
@@ -0,0 +1,45 @@
+// transpile :mocha
+
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import 'mochawait';
+import path from 'path';
+import AndroidBootstrap from '../../index';
+import ADB from 'appium-adb';
+import { errors } from 'mobile-json-wire-protocol';
+
+
+chai.should();
+chai.use(chaiAsPromised);
+
+describe('Android Bootstrap', function () {
+  let adb, androidBootstrap;
+  let rootDir = path.resolve(__dirname,
+                             process.env.NO_PRECOMPILE ? '../..' : '../../..');
+  const apiDemos = path.resolve(rootDir, 'test', 'fixtures', 'ApiDemos-debug.apk');
+  const systemPort = 4724;
+  before(async () => {
+    adb = await ADB.createADB();
+    const packageName = 'com.example.android.apis',
+          activityName = '.ApiDemos';
+    await adb.install(apiDemos);
+    await adb.startApp({pkg: packageName,
+                        activity: activityName});
```

Comment:
This is not to do with this code _per se_, but how come `adb.startApp` takes `activity` and `pkg`? It would be nicer to have both full (`activity` and `package`) or abbreviated (`act` and `pkg`) versions.